### PR TITLE
Staging API support

### PR DESCRIPTION
**Pull request by naira-cliqz**

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7081,7 +7081,7 @@
 					"-lxml2",
 					"$(inherited)",
 				);
-				OTHER_SWIFT_FLAGS = "-DMOZ_CHANNEL_RELEASE -DMOZ_TARGET_CLIENT -DPAID";
+				OTHER_SWIFT_FLAGS = "-DMOZ_CHANNEL_RELEASE -DMOZ_TARGET_CLIENT -DPAID -DBETA";
```
Comment (mahmoud-cliqz) [inline Client.xcodeproj/project.pbxproj]: @naira-cliqz  you don't need to add this as it is being added by fastlane anyway

File: Cliqz/Settings/CliqzAppSettingsTableViewController.swift [hunk 2]
```diff
@@ -30,8 +30,9 @@ class CliqzAppSettingsTableViewController: AppSettingsTableViewController {
 		// Account Settings
 		#if PAID
 		let accountSettings = generateAccountSettings(prefs: prefs)
-        let accountSettingsTitle = NSLocalizedString("Lumen Account", tableName: "Cliqz" , comment: "[Settings] Account")
-        settings += [SettingSection(title: NSAttributedString(string: accountSettingsTitle), children: accountSettings)]
+		let accountSettingsTitle = NSLocalizedString("Lumen Account", tableName: "Cliqz" , comment: "[Settings] Account")
```
Comment (mahmoud-cliqz) [inline Cliqz/Settings/CliqzAppSettingsTableViewController.swift]: you could just discard this commit

File: Client.xcodeproj/project.pbxproj (hunk 1 above)
Comment (naira-cliqz) [reply Client.xcodeproj/project.pbxproj]: Are you sure? :)

Comment (mahmoud-cliqz) [reply Client.xcodeproj/project.pbxproj]: yeah, in Fastfile you will find the following:
https://github.com/ghostery/browser-ios/blob/master/fastlane/Fastfile#L275

File: Cliqz/Settings/CliqzAppSettingsTableViewController.swift (hunk 2 above)
Comment (naira-cliqz) [reply Cliqz/Settings/CliqzAppSettingsTableViewController.swift]: I don't want to do all this rebase things with git :(

**Pull request merged**